pkgs/top-level/all-packages.nix

```diff
@@ -17979,7 +17979,9 @@ with pkgs;
 
   rpcs3 = libsForQt5.callPackage ../misc/emulators/rpcs3 { };
 
-  rstudio = libsForQt5.callPackage ../applications/editors/rstudio { };
+  rstudio = libsForQt5.callPackage ../applications/editors/rstudio {
+    boost = boost166;
+  };
```

Comment:
Versions should be overridden in `all-packages.nix`, like this. It makes it easier to spot and remove outdated overrides.

Reply:
Thanks, much clearer.